thing-url-adapter.js

```diff
@@ -405,8 +405,8 @@ class ThingURLDevice extends Device {
 
     if (this.scheduledUpdate) {
       clearTimeout(this.scheduledUpdate);
-    }
-    this.scheduledUpdate = setTimeout(() => this.poll(), POLL_INTERVAL);
+    }    
+    this.scheduledUpdate = setTimeout(() => this.poll(), this.adapter.poolInterval || POLL_INTERVAL);
```

Comment:
pollInterval is going to be in seconds in the config, so you'll need to multiply by 1000 here.

Reply:
> pollInterval is going to be in seconds in the config, so you'll need to multiply by 1000 here.

here or in loadThingURLAdapter ? actually we don't need to multiply every rescheduling